webapp/channels/src/components/admin_console/__snapshots__/schema_admin_settings.test.tsx.snap

```diff
@@ -150,7 +150,7 @@ exports[`components/admin_console/SchemaAdminSettings should match snapshot with
                 id="admin.true"
               />
             }
-            value={true}
```

Comment:
I updated these tests blindly in the last PR, and they were telling us about the bug :facepalm: